dns/Makefile

```diff
@@ -16,6 +16,7 @@ SHELL := /usr/bin/env bash
 CANARY_ZONES := canary.k8s.io. canary.kubernetes.io. canary.x-k8s.io. canary.k8s-e2e.com.
 PROD_ZONES := k8s.io. kubernetes.io. x-k8s.io. canary.k8s-e2e.com.
 ZONE_CONFIGS := zone-configs
+DOCKER_IMAGE := $(USER)/octodns
```

Comment:
We should the promoted image `us.gcr.io/k8s-artifacts-prod/infra-tools/octodns`.

Reply:
Changed, but I had to update `cloudbuild.yaml` to include `latest` tag